provisioning/transport/mqtt/test/_mqtt_test.js

```diff
@@ -250,6 +255,32 @@ describe('Mqtt', function () {
 
     });
 
+    ['$rid=123&retry-after=456&something=89','retry-after=456&$rid=123&something=89','retry-after=456&something=89&$rid=123'].forEach(function(theQueryString) {
+      it ('can parse response with multiple query parameters', function(callback) {
```

Comment:
>response [](start = 21, length = 8)

nit: reponse _topic_

Reply:
Done.

---
In reply to: [280188237](https://github.com/Azure/azure-iot-sdk-node/pull/549#discussion_r280188237) [](ancestors = 280188237)